.github/workflows/ci.yaml

```diff
@@ -17,14 +17,24 @@ concurrency:
 env:
   CARGO_ARGS: --no-default-features --features stdlib,zlib,importlib,encodings,sqlite,ssl
   # Skip additional tests on Windows. They are checked on Linux and MacOS.
+  # test_argparse: UnicodeDecodeError
+  # test_glob: many failing tests
+  # test_io: many failing tests
+  # test_os: many failing tests
+  # test_pathlib: support.rmtree() failing
+  # test_posixpath: OSError: (22, 'The filename, directory name, or volume label syntax is incorrect. (os error 123)')
+  # test_unicode: AttributeError: module '_winapi' has no attribute 'GetACP'
+  # test_venv: couple of failing tests
   WINDOWS_SKIPS: >-
+    test_argparse
```

Comment:
I am not getting how this one and test_rlcompleter was not tested or didn't fail on windows.

Reply:
It was not tested as it was considered platform independent

Reply:
Thanks. Is this currently failing on windows?
Then this test looks like a platform-dependent tests. it is expected to be not listed on platform dependent tests.
If the test success and it doesn't have platform-dependent tests, we don't need to add it here.
We only add tests here when the tests are not annotated well, so running it on Windows fails.
This list is expected to be removed in future once we annotate every tests enough

Reply:
See also #3960

Reply:
Yes this fails due to #5542.

Reply:
Thank you for confirming. Then please remove it from platform-independent tests list too